### PR TITLE
Mixed6

### PR DESCRIPTION
**Pull request by nouiz**

### CODE REVIEW

**Comment by abergeron** [comment]
There is a flake8 problem.

**Comment by nouiz** [comment]
fixed

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by slefrancois** [comment]
jenkins test this

**Comment by nouiz** [comment]
I updated this PR.

File: theano/misc/do_nightly_build [hunk 1]
```diff
@@ -36,6 +36,7 @@ if [ "$1" == "--buildbot" ]; then
     NOSETESTS=${ROOT_CWD}/Theano/bin/theano-nose
     export PYTHONPATH=${ROOT_CWD}:$PYTHONPATH
 else
+    ROOT_CWD=.
```
Comment (nouiz) [inline theano/misc/do_nightly_build]: Yes. When we call it manually as I does, it crash otherwise.

**Pull request merged**